tooling/lint-package-names.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+PACKAGES=$(echo packages/node_modules/{*,@ciscospark/*} | xargs -n 1 | sed 's/packages\/node_modules\///' | xargs -n 1 | grep -v '^@ciscospark$')
+for PACKAGE in $PACKAGES; do
```

Comment:
Should we also check to make sure there are no duplicate names? Although, I don't know how that would be possible.